benchmarks/oss.py

```diff
@@ -91,6 +94,7 @@ def train(
         optimizer = ddp.optimizer
         model = ddp
     else:
+        model = DDP(model, device_ids=[rank])
```

Comment:
should we toggle find_unused_parameters to True/False and see its impacts?

Reply:
ahh, good point ! it was not yet up for review actually, looks like there's something wrong with SDP, looking into that :) I'll test this option

Reply:
oops, sorry, didn't notice that! :-)